dev-python/nltk/nltk-3.4.5.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_6 )
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
I committed this with a fix to missing test dep.

Reply:
Awesome, thank you!